shared/reducers/fs.js

```diff
@@ -116,6 +117,25 @@ const updatePathItem = (
   }
 }
 
+const notifyDiskSpaceStatus = (diskSpaceStatus: Types.DiskSpaceStatus) => {
+  switch (diskSpaceStatus) {
+    case 'error':
+      NotifyPopup('Sync Error', {
+        body: 'You are out of disk space. Some folders could not be synced.',
+        sound: true,
+      })
+      break
+    case 'warning':
+      // TODO: propogate correct number of GB from setting
+      NotifyPopup('Disk Space Low', {body: 'You have less than 1 GB of storage space left.'})
+      break
+    case 'ok':
+      break
+    default:
+      Flow.ifFlowComplainsAboutThisFunctionYouHaventHandledAllCasesInASwitch(diskSpaceStatus)
+  }
+}
```

Comment:
The reducer doesn't seem like the right place for this. Can we move it into Saga?

Reply:
You're right, I'll move it.